src/components/Card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Card extends React.Component {
+
+  render() {
+    return (
+      <div className={ this.props.modifier ? 'p-card--' +  this.props.modifier : 'p-card' }>
```

Comment:
Don't know how you feel about template literals, but I find them easier to read in ternary operators.

``this.props.modifier ? `p-card--${this.props.modifier}` : 'p-card'``

Reply:
I'm all over template literals 😍